api/routes.go

```diff
@@ -31,11 +31,9 @@ func (api *API) buildHTTPRoutes() {
 	api.staticRouter.GET("/user/limits", api.WithDBSession(api.noValidate(api.userLimitsGET)))
 	api.staticRouter.GET("/user/stats", api.WithDBSession(api.validate(api.userStatsGET)))
 	api.staticRouter.GET("/user/uploads", api.WithDBSession(api.validate(api.userUploadsGET)))
-	api.staticRouter.DELETE("/user/uploads/:uploadId", api.WithDBSession(api.validate(api.userUploadDELETE)))
+	api.staticRouter.DELETE("/user/uploads/:skylink", api.WithDBSession(api.validate(api.uploadsDELETE)))
```

Comment:
should you rename `uploadsDELETE` to `userUploadsDELETE` to keep this endpoint name in line with other user endpoints ?

Reply:
Yes, that makes sense. Changed.